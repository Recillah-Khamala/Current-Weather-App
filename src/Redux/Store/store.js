import { ConfigureStore } from '@reduxjs/toolkit';

const Store = ConfigureStore({
  reducer: {},
});

export default Store;
