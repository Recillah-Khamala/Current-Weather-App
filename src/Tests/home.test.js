import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { HashRouter as Router } from 'react-router-dom';
import Store from '../Redux/Store/store';
import Home from '../Pages/home';

describe('Testing by jest snapshots', () => {
  it('matches home  Snapshot', () => {
    const homeDisplay = renderer
      .create(
        <Provider store={Store}>
          <Router>
            <Home />
          </Router>
        </Provider>,
      )
      .toJSON();
    expect(homeDisplay).toMatchSnapshot();
  });
});
