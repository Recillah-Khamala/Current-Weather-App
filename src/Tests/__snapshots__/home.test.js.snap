// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing by jest snapshots matches home  Snapshot 1`] = `
<div>
  <section
    className="text-white relative"
  >
    <h2
      className="mt-8 mb-12 lg:mb-2 text-white text-3xl lg:text-4xl lg:tracking-wider lg:font-bold font-semibold relative container pt-2 lg:pt-none px-4 mx-auto lg:mx-none text-center"
    >
      Current Weather App
    </h2>
    <img
      alt="World Map"
      className="w-full pt-4 md:w-auto opacity-20 lg:px-44"
      src="world_weather_map-removebg-preview.png"
    />
  </section>
  <div
    className="text-white bg-strip-blue p-1 mt-12"
  >
    <p
      className="text-4xl font-bold uppercase leading-6 text-center py-6 mb-none "
    >
      Check Weather
      

       
      anywhere you are
    </p>
  </div>
</div>
`;
